integration/listener_test.go

```diff
@@ -34,12 +30,14 @@ func (suite *serviceBusSuite) TestListenWithDefault() {
 			suite.TopicName,
 		)
 		suite.True(contextCanceledError(err), "listener listen function failed", err.Error())
-		suite.EqualValues(listener.subscriptionEntity.Name, "default")
+		suite.EqualValues(listener.Subscription().Name, "default")
```

Comment:
expose ther servicebus entities via a func.
moving the tests in a different package showed the need for these apis.